docs/root/configuration/http/http_filters/ext_authz_filter.rst

```diff
@@ -106,16 +106,18 @@ In this example we add additional context on the virtual host, and disabled the
     virtual_hosts:
     - name: local_service
       domains: ["*"]
-      per_filter_config:
+      typed_per_filter_config:
```

Comment:
Did you build a list of all deprecated fields to grep for? Just curious how your script works. Can you share it in a branch or gist? It's fine if it's super-hacky, just want to get a better idea of what I should be trying to validate.

Reply:
Deprecated fields in examples were updated by hand (I did build a list of deprecated fields via grep first). I can share the script I used for updating v2 messages/fields/types/enums to v3 ones though.

Reply:
Gist with the script (pretty hacky!): https://gist.github.com/dmitri-d/e5957de963d7e51004f4fce2ba82a383

Reply:
See my other comment about how this can be less manual in the future, but if there is anything we should commit for future use let's definitely do that.

Reply:
@dmitri-d is there some place you grepped out a list of all the deprecated field names and then checked to see if any still exist in docs? I realize there could be many false positives, but this would give me confidence that we haven't missed any (dot-the-Is, cross-the-Ts).